MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -13,6 +13,8 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		didSet { tableView.reloadData() }
 	}
 
+	@IBOutlet var errorView: ErrorView!
```

Comment:
This property can be private
```suggestion
	@IBOutlet private var errorView: ErrorView!
```